### ISSUE TITLE
Should mujoco_py be in environment.yml by default?

### ISSUE THREAD

**Issue by joaospinto** [openai/rllab#162]
Hi,

I started using rllab recently. It seems that running ./scripts/setup_{osx,linux}.sh fails if you don't have MuJoCo installed, and I had to comment out the corresponding line in environment.yml to get it to work. I know this is a bit of a nitpick, but either this entry should be removed or the documentation should mention that, if you don't own a MuJoCo license, you should edit environment.yml first; it's a bit odd that one would assume that a proprietary package, however popular, is installed.

Thanks,
Joao

**Comment by dementrock** [comment]
Yes it makes sense. Sorry I've been extremely busy lately. Could you submit a pull request for this?
